### PR TITLE
monasca: Set proper log-level for agent

### PR DESCRIPTION
**Pull request by matrixik**

### CODE REVIEW

File: chef/cookbooks/monasca/recipes/agent.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@
     monasca_api_url: monasca_api_url,
     agent_dimensions: agent_dimensions,
     log_dir: agent_settings["log_dir"],
-    log_level: agent_settings[:debug] ? "DEBUG": "INFO",
+    log_level: agent_settings[:debug] ? "DEBUG" : agent_settings[:log_level],
```
Comment (toabctl) [inline chef/cookbooks/monasca/recipes/agent.rb]: Why do we need ```agent_settings[:debug]```  at all if we have ```agent_settings[:log_level]``` ?

Comment (matrixik) [inline chef/cookbooks/monasca/recipes/agent.rb]: Not sure, I think it appeared at the beginning to help with development. I will remove it if you want .

Comment (toabctl) [inline chef/cookbooks/monasca/recipes/agent.rb]: yes. please clean that up

Comment (matrixik) [inline chef/cookbooks/monasca/recipes/agent.rb]: Done

**Pull request merged**